setup.py

```diff
@@ -1,6 +1,7 @@
 from setuptools import setup, find_packages
 from aztk import constants
 
+
 setup(name='aztk',
       version='0.1',
```

Comment:
version rev?

Reply:
yeah, this should be handled the same way we handle the cli versioning (reading the aztk_sdk/version.py file).